paddleseg/core/train.py

```diff
@@ -208,7 +208,10 @@ def train(model,
                     edges=edges)
                 loss = sum(loss_list)
                 loss.backward()
-                optimizer.step()
+                if isinstance(optimizer, paddle.optimizer.lr.ReduceOnPlateau):
```

Comment:
Agree with the reviewer @juncaipeng, please add a little bit comments in English there, otherwise other users might not understand the new code. Thanks.

Reply:
ok i will